source/reference/editor-mode.txt

```diff
@@ -1,8 +1,8 @@
 .. _mongosh-editor-mode:
 
-===========
-Editor Mode
-===========
+==========================
+Use an Editor for Commands
```

Comment:
[praise]

This title change makes everything clearer for the user. Excellent choice.